### DISCUSSION TITLE
Expiry time of Magic Links

### DISCUSSION THREAD

**Discussion by cacoos** [supabase/supabase#16194] [Questions]
Hey!

I just want to know what I am doing wrong. I have set a long expiry time for OTP mails.

<img width="400" alt="Screenshot 2023-08-03 at 07 28 14" src="https://github.com/supabase/supabase/assets/30879716/407b59ee-69e4-470d-b760-6fb65efe508c">

But after 10-15 minutes, the link inside the mail doesn't work.

In supabase logs, I see this error:
`"api","error":"400: No associated flow state found. 400: Flow state is expired","level":"info","method":"GET","msg":"400: No associated flow state found. 400: Flow state is expired"`

Watching the logs I can confirm that is nothing related to limitations (https://supabase.com/docs/guides/auth/auth-email-templates#limitations). The email platform (Gmail) isn't clicking the URLs.

Am I overlooking something?

**Comment by bombillazo** [comment]
Any answer to this?
